recipe/meta.yaml

```diff
@@ -9,47 +9,44 @@
 {% set build = build + 100 %}
 {% endif %}
 
+{% set mkl = "<2024" %}
+
 package:
-  name: libtorch
+  name: libtorch-split
```

Comment:
Let's not do this unless absolutely necessary. I was okay doing this because of libmagma builds, but now that they are gone, let's not do this split build and keep the top level as `libtorch`.

Reply:
Do you prefer me reverting it as more commits, or rebasing the whole thing?

Reply:
Whichever is easier for you

Reply:
Ok, I went for more commits, because I wanted to retain full history and attribution, and collapsing the code back seems relatively clean anyway. I'm going to finish testing in a few minutes and push.